chaco/polygon_plot.py

```diff
@@ -144,8 +143,8 @@ def hittest(self, screen_pt, threshold=7.0, return_distance=False):
     # Event handlers
     #------------------------------------------------------------------------
 
-    @on_trait_change('edge_color, edge_width, edge_style, face_color, alpha')
-    def _attributes_changed(self):
+    @observe('edge_color, edge_width, edge_style, face_color, alpha')
+    def _attributes_changed(self, event):
```

Comment:
same comment as above - about renaming this to `_invalidate_and_redraw` if such a method doesn't already exist in the class heirarchy